docs/api.md

```diff
@@ -12,13 +12,32 @@ The peer capability label is `tailscale.com/cap/secrets`.
 Calls to the API must include a header `Sec-X-Tailscale-No-Browsers: setec`.
 This prevents browser scripts from initiating calls to the service.
 
+
 ## HTTP Status
 
 - Invalid request parameters report 400 Invalid request.
 - Access permission errors report 403 Forbidden.
 - Requests for unknown values report 404 Not found.
 - All other errors report 500 Internal server error.
 
+
+## Permissions
+
+The service defines named _actions_ that are subject to access control:
+
+- `get`: Denotes permission to fetch the contents of a secret.                                                                             |
```

Comment:
With only "get", can I get version numbers?

Reply:
You cannot _list_ version numbers, but you can request a specific version.